_projects/2025/100528259/100528259.Rmd

```diff
@@ -0,0 +1,986 @@
+---
+title: "An 8-Bit Look at the Evolution of BMI Values of Olympic Athletes"
+description: |
+  This study displays the evolution of BMI values of Olympic athletes who competed in the Summer Olympic Games of 1960 and 2016.
+categories: "2024"
+author: Gür Piren
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
+```
+
```

Comment:
Please add a bit of introduction where you show us the selected graph (include a png in the PR and highlight it as e.g. David does [here](https://github.com/IBiDat/dataviz/blob/9a3bc0367b1c0b297765b59ff6f3c34cef06a204/_projects/2024/100535712/100535712.Rmd#L26)), and tell us why you chose it. Also, please link the original source.